src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -179,24 +185,36 @@ public async Task<bool> UpdateAsync(string userId, string roleId)
             return true;
         }
 
+        private List<UserReleaseRole> GetUserReleaseRoles(string userId)
+        {
+            var result = new List<UserReleaseRole>();
+            result.AddRange(_contentDbContext.UserReleaseRoles.Where(x => x.UserId == Guid.Parse(userId)).Select(x => new UserReleaseRole
```

Comment:
I'm not sure why you result.AddRange here to an empty list, why not just return the result of the query?